t/29-backend-ipmi.t

```diff
@@ -36,18 +37,17 @@ $ipmi->redefine(ipmitool => sub { $_[1] =~ /power status/ ? $ipmitool_mock->ipmi
 ok $backend->restart_host, 'can call restart_host';
 $ipmi->noop('ipmitool');
 my $distri = Test::MockModule->new('distribution');
-$distri->redefine(add_console => sub {
-        my $ret = Test::MockObject->new();
-        $ret->set_true('backend');
-        return $ret;
-});
 $testapi::distri = distribution->new;
 
 ok $backend->do_start_vm, 'can call do_start_vm';
 ok $backend->do_stop_vm, 'can call do_stop_vm';
 ok !$backend->check_socket(undef), 'check_socket not returning true by default';
 ok $backend->get_mc_status, 'can call get_mc_status';
 
+is $testapi::distri->{consoles}->{sol}->{args}->{log}, '1';
+$testapi::distri->{consoles}->{sol}->{DISPLAY} = "display";
+ok !$testapi::distri->{consoles}->{sol}->callxterm('ipmi', "console"), "can create console with log enabled";
```

Comment:
This fails in CI now with

```
8: 
8: #   Failed test at ./29-backend-ipmi.t line 47.
8: #          got: '0'
8: #     expected: '1'
8: Missing 'xterm-console' at /opt/consoles/localXvnc.pm line 37.
8: # Tests were run but no plan was declared and done_testing() was not seen.
8: # Looks like your test exited with 2 just after 14.
8: [13:31:37] ./29-backend-ipmi.t ...................... 
8: Dubious, test returned 2 (wstat 512, 0x200)
8: Failed 1/14 subtests 
```

because "xterm-console" is not installed in the development environment. I suggest you add the package "xterm-console" in dependencies.yaml after line 91 in the test requirements and call `make update-deps` and submit the changes as well